std/internal/math/gammafunction.d

```diff
@@ -531,8 +548,17 @@ unittest {
 
 
 private {
-enum real MAXLOG = 0x1.62e42fefa39ef358p+13L;  // log(real.max)
-enum real MINLOG = -0x1.6436716d5406e6d8p+13L; // log(real.min*real.epsilon) = log(smallest denormal)
+static if (floatTraits!(real).realFormat == RealFormat.ieeeExtended) {
+    enum real MAXLOG = 0x1.62e42fefa39ef358p+13L;  // log(real.max)
+    enum real MINLOG = -0x1.6436716d5406e6d8p+13L; // log(real.min_normal*real.epsilon) = log(smallest denormal)
+}
+else static if (floatTraits!(real).realFormat == RealFormat.ieeeDouble) {
+    enum real MAXLOG = 0x1.62e42fefa39efp+9L;  // log(real.max)
+    enum real MINLOG = -0x1.74385446d71c3p+9L; // log(real.min_normal*real.epsilon) = log(smallest denormal)
```

Comment:
Please replace with this lines. The `MINLOG` was changed, the MAXLOG remains the same.
```D
    enum real MAXLOG =  0x1.62e42fefa39efp+9; /// log(2^+1024) at Wolfram Alpha, 21.07.2015
    enum real MINLOG = -0x1.74910d52d3052p+9; /// log(2^-1075) at Wolfram Alpha, 21.07.2015
```